cmd/step-ca/main.go

```diff
@@ -117,7 +117,7 @@ func main() {
 	app.HelpName = "step-ca"
 	app.Version = step.Version()
 	app.Usage = "an online certificate authority for secure automated certificate management"
-	app.UsageText = `**step-ca** <config> [**--password-file**=<file>]
+	app.UsageText = `**step-ca** [config] [**--context**=name] [**--password-file**=<file>]
```

Comment:
Add `<x>` to be consistent:
```suggestion
	app.UsageText = `**step-ca** [config] [**--context**=<name>] [**--password-file**=<file>]
```